Assets/HoloToolkit/UI/Shaders/3DTextShader.shader

```diff
@@ -70,20 +72,22 @@
 
                 v2f vert (appdata_t v)
                 {
+                    UNITY_SETUP_INSTANCE_ID(v);
                     v2f o;
                     o.vertex = mul(UNITY_MATRIX_MVP, v.vertex);
                     o.texcoord = TRANSFORM_TEX(v.texcoord, _MainTex);
                     o.color = v.color;
                     #ifdef UNITY_HALF_TEXEL_OFFSET
                     o.vertex.xy += (_ScreenParams.zw-1.0)*float2(-1,1);
                     #endif
+                    UNITY_INITIALIZE_VERTEX_OUTPUT_STEREO(o);
                     return o;
                 }
 
                 half4 frag (v2f i) : COLOR
                 {
                     half4 col = i.color;
-                    col.a *= tex2D(_MainTex, i.texcoord).a;
+                    col.a *= tex2D(_MainTex, UnityStereoScreenSpaceUVAdjust(i.texcoord, _MainTex_ST)).a;
```

Comment:
What is a bit confusing is that I think the single-pass rendering that the hololens uses may or may not have the same kinds of requirements. There are two different defines for single-pass:  UNITY_STEREO_INSTANCING_ENABLED and UNITY_SINGLE_PASS_STEREO. 

The stereo instancing method doesn't use a packed render texture, but instead outputs to a render target array. Those macros and the linked documentation seem to pertain more to scaling and transforming uv's to the left or right eye target position which isn't necessary on the hololens.